Tools/Scripts/webkitpy/benchmark_runner/benchmark_runner.py

```diff
@@ -85,6 +92,36 @@ def available_plans():
         plans = [os.path.splitext(plan_file)[0] for plan_file in os.listdir(BenchmarkRunner.plan_directory()) if plan_file.endswith(".plan")]
         return plans
 
+    @staticmethod
+    def format_subtests(subtests):
+        return ['{}/{}'.format(subtest['suite'], subtest['test']) if subtest['suite'] else subtest['test'] for subtest in subtests]
+
+    @staticmethod
+    def available_subtests(cls, plan_file):
+        plan = BenchmarkRunner._load_plan_data(plan_file)[1]
+        if 'subtests' not in plan:
+            return []
+        return BenchmarkRunner.format_subtests(plan['subtests'])
+
+    def validate_subtests(self, subtests):
+        valid_subtests = self._plan['subtests']
+        subtests_to_run = []
+        for subtest in subtests:
+            if '/' in subtest:
```

Comment:
I'm conflicted on that. On one hand it'd be nice for QoL, but on the other if the user doesn't explicitly specify a suite name we should default to it being a test name (assuming we decide that it'd be best to infer a suite name when one isn't passed in). So on the off chance that a suite and a test both have the same name, it'll default to just running the one subtest instead of everything from the suite. Unless maybe it'd be a better to reverse this logic, and check suites first before individual subtests?